docs/installation.rst

```diff
@@ -47,5 +69,19 @@ Backward Compatibility
 
 Older versions than ones listed above are unsupported. These oldest versions remain tested in `Travis-CI` and deployment
 procedure for traceability, but are not guaranteed to work, nor provide all functional or security features and will not
-be actively maintained. If you identify a easy fix for such an older version, please submit an `issue`_ to be considered
-for integration. It is greatly recommended to upgrade your Python version to receive all applicable security patches.
+be actively maintained. If you identify an easy fix for such an older version, please submit an `issue`_ to be
+considered for integration. It is greatly recommended to upgrade your Python version to receive all applicable security
+patches.
+
+Installation for Twitcher
```

Comment:
Would it make sense to mention the initialization of magpie db permissions before starting twitcher the first time?

Reply:
Yes. I can add that.